src/pages/nfts/profile/[accountAddress].tsx

```diff
@@ -14,9 +14,13 @@ const NftProfilePage = () => {
   const isConnectedProfile = account?.toLowerCase() === accountAddress?.toLowerCase()
   const {
     profile: profileHookState,
-    isFetching: isProfileFetching,
+    isValidating: isProfileFetching,
     refresh: refreshProfile,
-  } = useProfileForAddress(accountAddress)
+  } = useProfileForAddress(accountAddress, {
```

Comment:
Have to revalidate since data (profile) that used in useNftsForAddress uses useSwr